### ISSUE TITLE
Open Tab From Context Menu

### ISSUE THREAD

**Issue by etmiranda** [cmderdev/cmder#983]
Apologies if this is documented elsewhere, I looked but didn't see anything. I tried updating to 1.3.0 in the hopes that there was a fix, but it's still not working for me.

What I want:
Right click context menu to open Cmder
If Cmder is already open, opens new tab at context menu location
Otherwise, opens Cmder with one tab at context menu location

I have tried using Settings > Appearance > Single instance mode, but when I try to open a location it just re-focuses Cmder. No new tab, not even a new path, just re-focuses the existing tab.

I have tried a registry command to do:
C:\Cmder\Cmder.exe /single "%V"
But this opens a new tab at the previous tab's location, not my context menu location.

Is there a way to get this working, or is there a bug?

(Also, it'd be nice if you could drag windows/tabs to merge or separate them, like the Chrome browser can do, but I realize that's probably tricky.)

**Comment by rlabrecque** [comment]
This seems to be a bug based on #356

**Comment by Hugo-KTM** [comment]
I am pretty sure this _is_ #356 and contrary to the current status of #356 this is not fixed for version 1.3.0.

I edited `vendor/init.bat` to print `%CD%`, `%CMDER_START%` and `%*` and all output shows that those variables do not change during a Cmder session. It looks like `Cmder.exe /SINGLE` is reusing the environment of the first session i. e., since `Cmder.exe` does not stay in memory, it looks like `/SINGLE` searches for a running `ConEmu` session (looks like it chooses the one with the lowest PID) and just reuses the existing environment of that session.

logging snipped:
```
setlocal
for /f "delims=" %%# in ('powershell get-date -format "{yyyy-MM-dd HH:mm:ss,fff}"') do @set TIMESTAMP=%%#
:: if NOT [%*] == [] 
echo %TIMESTAMP%   %%CD%%:"%CD%"   %%CMDER_START%%:%CMDER_START%   %%*:%*>> %CMDER_ROOT%\vendor\init.log
endlocal
```

**Comment by Jackbennett** [comment]
Pointing more loose ends back to #420
